app/lib/ability.rb

```diff
@@ -58,8 +67,32 @@ def initialize(user)
     can :read, Organization, id: accessible_groups.pluck(:id)
     can :read, Site, id: accessible_groups.pluck(:id)
 
-    # Anyone can manage clients and client data in the groups they can access
-    can :manage, Client, vita_partner: accessible_groups
+    # This was overly permissive. We should work out what the permissions should
+    # be for each role and reduce this check. As we need to modify this, please
+    # break out the role and specify permissions more granularly
```

Comment:
I don't think this comment is applicable anymore!

Reply:
The comment was guidance for folks who have to come in and manipulate this part of the file in the future. If you feel it's still unnecessary with the additional context I can cut it out. The idea that I was hoping for is that as people come in and touch it more of these can be broken out into their own role specific permission whitelists.